content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
+---
+
+{{% pageinfo %}}
+Use this document with our [Cobalt API documentation](https://docs.cobalt.io) to
+manage assets and pentests.
+{{% /pageinfo %}}
+
+<!-- Future task: set up variables for `YOUR-PERSONAL-API-TOKEN` and
+`YOUR-V2-ORGANIZATION-TOKEN`. May support automated populating of REST calls. -->
+
+This page describes how you can use our API To create an [asset](../../getting-started/glossary/#asset). 
+
+## Create an API Token in the Cobalt UI
+
+1. Navigate to https://app.cobalt.io/settings/api-tokens.
+1. If needed, sign in to the app.
+1. Enter an API Token Name.
+1. Select Generate New Token.
+1. In the modal that appears, you should see your API Token, in the **Secret
+   Token** text box.
+
+Save the API Token. After you exit the modal, you won't see the full token again.
+If you lose it, you may have to revoke the token and start over.
```

Comment:
Save the API token. After you close the overlay, you won't see the full token again.
If you lose it, you may have to revoke the token and start again.

Reply:
Will accept. I searched for instances of both, and "close the overlay" is 4x more popular than "exit the modal".